ajax/libs/exceljs/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "exceljs",
+  "version": "0.8.0",
+  "filename": "exceljs.browser.min.js",
+  "description": "Excel Workbook Manager - Read and Write xlsx and csv Files.",
+  "license": "MIT",
+  "author": "Guyon Roche",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/guyonroche/exceljs.git"
+  },
+  "keywords": [
+    "xlsx",
+    "json",
+    "csv",
+    "excel",
+    "font",
+    "border",
+    "fill",
+    "number",
+    "format",
+    "number format",
+    "alignment",
+    "office",
+    "spreadsheet",
+    "workbook",
+    "defined names",
+    "data validations",
+    "rich text",
+    "in-cell format",
+    "outlineLevel",
+    "views",
+    "frozen",
+    "split",
+    "pageSetup"
+  ],
+  "autoupdate": {
```

Comment:
Please use npm auto-update instead. You can find how this library is used in https://github.com/guyonroche/exceljs#browser. Which files users need in https://github.com/guyonroche/exceljs#prebundled. And you can find then in npm package. 

Please remember to exclude `es5` in some versions. Files under it are for back-end.
pattern is like
```
"exceljs*",
"!(es5)/**/*"
```